experimental/origin-dapp2/src/pages/_ListingBadge.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+const ListingBadge = ({ status, featured }) => {
+  if (status === 'sold') {
+    return <div className="badge badge--sold">sold</div>
+  }
+  // TODO: Unavailable (multi unit, pending)
+  // TODO: Pending (single unit, pending)
+  // TODO: Sold Out (multi unit, no units)
+  if (featured) {
+    return <div className="badge badge--featured">featured</div>
+  }
+
+  return null
+}
+
+export default ListingBadge
+
+require('react-styl')(`
+  .badge
+    background: gray
+    color: white
+
+    &--sold
```

Comment:
Are these colors OK? I only saw the featured one in the screenshots.

Reply:
I think pending should be `var(--dark)`

Reply:
I think the intent was for `Sold`, `Sold Out`, `Unavailable`, and `Pending` to all be the same color (`--dark`). They equate to a similar UX, which is that a buyer is unable to make a new offer.

Reply:
Makes sense! Will make that change.